### PR TITLE
Feat/internationlisation

### PR DESCRIPTION
**Pull request by gautism1**

### CODE REVIEW

**Review by smilegupta — COMMENTED** [review]
Looks good to me from functionality wise.

One suggestions from Design Perspective:
Make the button size small of english and espnola and align them on right side. It will look more good.

**Comment by rparsh** [comment]
@gautism1 Move the language buttons inside sidebar(bottom) [hide from mobile for now]

**Review by smilegupta — APPROVED** [review]

**Comment by rparsh** [comment]
@gautism1 default language is english but UI render French as default

File: src/i18n/wrapper.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+function Wrapper(props) {
+    const [locale, setLocale] = useState(local);
+    const [message, setMessage] = useState(lang);
+
+    function selectLang(e) {
+        const newlocale = e.target.value;
+
+        setLocale(newlocale);
+        if (newlocale === "en-US") {
```
[the remark this answers is not in the document]
Comment (gautism1) [reply src/i18n/wrapper.js:25]: Sir the default is English and it renders English as default  ,also i change the newlocale condition to .include("en-")
Thanks,

**Comment by gautism1** [comment]
Resolves conflicts  and merged internationalisation and data-dynamic branches

**Comment by rparsh** [comment]
@gautism1 please revert data dynamic branch merge ...that branch is not merged to master you cannot merge it directly

**Review by rparsh — CHANGES_REQUESTED** [review]
Revert dtaa dynamic branch

**Pull request merged**